packages/live-share-canvas/src/core/Internals.ts

```diff
@@ -203,10 +203,12 @@ export function isRectangleInsideRectangle(r: IRect, containingRectangle: IRect)
     const bottomLeft = { x: r.left, y: r.bottom };
     const bottomRight = { x: r.right, y: r.bottom };
 
-    return isPointInsideRectangle(topLeft, containingRectangle) &&
+    return (
```

Comment:
Why?? The parentheses serve no purpose

Reply:
don't think this is configurable either

Reply:
This is configurable; adding it to the re-vote